docs/index.md

```diff
@@ -1,3 +1,15 @@
 # Data Generation Toolkit Documents Library
 
-This is sample text to make sure the Github pages setup is working.
+This site will be the home for published documents by the Data Generation Toolkit project (at least until we decide to use something different). Currently this is placeholder content and does not need to be kept (although the directions below should probably be migrated to some other location if they aren't kept on the index page or if the index page gets really big).
+
+## To add content to this site:
+
+1. Clone the [Data Generation Toolkit])(https://github.com/SFDO-Community-Sprints/DataGenerationToolkit) repository.
+1. Create a new branch in your local copy for your new work: `git switch -c documentation/my-new-content` (please try to use something more descriptive than "my-new-content" you are looking for a short description that helps others know what to expect).
```

Comment:
As much as I prefer switch -c, didn't it seem that hardly anyone had the appropriate version of git installed yet?

Reply:
Replaced `switch` with `branch`